testutil/testutil.go

```diff
@@ -92,6 +92,52 @@ func FaultOrPanicToErr(f func()) (err error) {
 	return err
 }
 
+// Contains fails the test if needle is not contained within haystack, if haystack or needle is
+// an empty slice, or if needle is longer than haystack.
+func Contains(tb testing.TB, haystack, needle []string) {
```

Comment:
Shouldn't this be.... generic? ;p

Reply:
Otherwise, let's use `ContainsStringSlice` function name?

Reply:
You're right! Something generic would suit here. Let me try to tackle that in next PR. Changing name for now! 🙂